extension/chrome/elements/compose.ts

```diff
@@ -167,8 +167,14 @@ export class ComposeView extends View {
     if (this.replyMsgId) {
       await this.renderModule.fetchReplyMeta(Object.keys(storage.sendAs!));
     }
-    if (this.isReplyBox && this.threadId && !this.ignoreDraft && storage.drafts_reply && storage.drafts_reply[this.threadId]) {
-      this.draftId = storage.drafts_reply[this.threadId]; // there may be a draft we want to load
+    if (this.isReplyBox) { // reply
+      if (this.threadId && !this.ignoreDraft && storage.drafts_reply && storage.drafts_reply[this.threadId]) {
+        this.draftId = storage.drafts_reply[this.threadId]; // there may be a draft we want to load
+      }
+    } else { // compose
+      if (!this.draftId) {
+        this.draftId = this.draftModule.localDraftPrefix;
```

Comment:
Now I see this may be intentional

Reply:
Maybe we could then be more explicit about it, and have a `this.draftModule.localNewMessageDraftId` also, to make it clear there is only one such new message draft possible

Reply:
Done :+1: